src/test/scala/com.snowplowanalytics.snowplow.badrows/SchemaValidationSpec.scala

```diff
@@ -83,9 +83,8 @@ object SchemaValidationSpec {
     CirceValidator.validate(decoded, schema.getOrElse(throw new RuntimeException(s"Schema could not be found: $schema")))
   }
 
-  // TODO: replace to actual Iglu Central once merged
-  val http = Registry.HttpConnection(URI.create("https://raw.githubusercontent.com/snowplow/iglu-central/release/r110/"), None)
-  val igluCentral = Registry.Http(Registry.Config("Iglu Central R110 PR (temporary)", 0, List("com.snowplowanalytics.snowplow.badrows")), http)
+  val http = Registry.HttpConnection(URI.create("https://raw.githubusercontent.com/snowplow/iglu-central/badrows_querystring/"), None)
```

Comment:
I afraid we'll forget it again.

Reply:
This is in a [temporary commit](https://github.com/snowplow-incubator/snowplow-badrows/pull/36/commits/4fda9eecc896661c37355983a978ac396fd36179) that will be removed when preparing the release